metals/src/main/scala/scala/meta/internal/metals/Messages.scala

```diff
@@ -105,6 +105,11 @@ object Messages {
     "Could not extract the given definition, please check the logs for more details or report an issue."
   )
 
+  val PasteWithIndentationFailed = new MessageParams(
+    MessageType.Error,
+    "Could not paste with indentation, please check the logs for more details or report an issue."
```

Comment:
Maybe we can be more specific and tell which file is relevant
```suggestion
    "Could not paste with indentation, please check the metals.log for more details or report an issue."
```